SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -96,6 +96,41 @@ FNetworkGUID USpatialPackageMapClient::GetNetGUIDFromEntityId(const Worker_Entit
 	return GetNetGUIDFromUnrealObjectRef(ObjectRef);
 }
 
+UObject* USpatialPackageMapClient::GetObjectFromUnrealObjectRef(const FUnrealObjectRef& ObjectRef)
+{
+	FNetworkGUID NetGUID = GetNetGUIDFromUnrealObjectRef(ObjectRef);
+	if (NetGUID.IsValid())
+	{
+		return GetObjectFromNetGUID(NetGUID, true);
+	}
+
+	return nullptr;
+}
+
+FUnrealObjectRef USpatialPackageMapClient::GetUnrealObjectRefFromObject(UObject* Object)
+{
+	if (Object == nullptr)
+	{
+		return SpatialConstants::NULL_OBJECT_REF;
+	}
+
+	FNetworkGUID NetGUID = GetNetGUIDFromObject(Object);
+
+	return GetUnrealObjectRefFromNetGUID(NetGUID);
+}
+
+UObject* USpatialPackageMapClient::GetObjectFromEntityIdAndComponentId(Worker_EntityId EntityId, Worker_ComponentId ComponentId)
+{
+	uint32 Offset = 0;
+	bool bFoundOffset = TypebindingManager->FindOffsetByComponentId(ComponentId, Offset);
```

Comment:
nit: Inverse this and just do the FindOffsetByComponentId in the if statment, and return if valid. Or put an UNLIKELY macro around the !bFoundOffset.